### PR TITLE
Feature: lighthouse configs

### PR DESCRIPTION
**Pull request by Wolmin**
Provided basic lighthouse.toml config files

### CODE REVIEW

File: testnet/lighthouse/lighthouse.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
+slots-per-restore-point = 32
```
Comment (mxmar) [inline testnet/lighthouse/lighthouse.toml:21]: We can remove it, this is relevant for explorers only.

File: mainnet/lighthouse/lighthouse.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
+slots-per-restore-point = 32
```
Comment (mxmar) [inline mainnet/lighthouse/lighthouse.toml:21]: We can remove it, this is relevant for explorers only.

File: devnets/3030/lighthouse/lighthouse.toml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
+slots-per-restore-point = 32
```
Comment (mxmar) [inline devnets/3030/lighthouse/lighthouse.toml:21]: We can remove it, this is relevant for explorers only.

File: testnet/lighthouse/lighthouse.toml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+enr-udp-port = 9000
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
```
Comment (mxmar) [inline testnet/lighthouse/lighthouse.toml:20]: Please use real testnet enrs: https://github.com/lukso-network/network-configs/blob/057a26ee38fa21b26e7be67e98912fb5a3d8016b/testnet/prysm/prysm.yaml#L19

File: mainnet/lighthouse/lighthouse.toml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+enr-udp-port = 9000
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
```
Comment (mxmar) [inline mainnet/lighthouse/lighthouse.toml:20]: For mainnet we can leave empty now. We will create PR when bootnodes are up and running :+1:

File: devnets/3030/lighthouse/lighthouse.toml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+enr-udp-port = 9000
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
```
Comment (mxmar) [inline devnets/3030/lighthouse/lighthouse.toml:20]: For devnets we can leave empty. We don't want to share them.

File: devnets/3030/lighthouse/deploy_block.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+0
```
Comment (mxmar) [inline devnets/3030/lighthouse/deploy_block.txt:1]: For mainnet and testnet `deploy_block.txt` is missing, it's `0` always.

**Review by mxmar — CHANGES_REQUESTED** [review]

File: testnet/lighthouse/lighthouse.toml (hunk 1 above)
Comment (Wolmin) [reply testnet/lighthouse/lighthouse.toml]: Done

File: mainnet/lighthouse/lighthouse.toml (hunk 2 above)
Comment (Wolmin) [reply mainnet/lighthouse/lighthouse.toml]: Done

File: devnets/3030/lighthouse/lighthouse.toml (hunk 3 above)
Comment (Wolmin) [reply devnets/3030/lighthouse/lighthouse.toml]: Done

File: testnet/lighthouse/lighthouse.toml (hunk 4 above)
Comment (Wolmin) [reply testnet/lighthouse/lighthouse.toml]: Done

File: devnets/3030/lighthouse/lighthouse.toml (hunk 6 above)
Comment (Wolmin) [reply devnets/3030/lighthouse/lighthouse.toml]: Done

File: mainnet/lighthouse/lighthouse.toml (hunk 5 above)
Comment (Wolmin) [reply mainnet/lighthouse/lighthouse.toml]: Done

File: devnets/3030/lighthouse/deploy_block.txt (hunk 7 above)
Comment (Wolmin) [reply devnets/3030/lighthouse/deploy_block.txt:1]: It was already there before this PR, so it is fine

**Review by mxmar — APPROVED** [review]
LGTM @Wolmin But you need to test `enr-address = "0.0.0.0"` (lighthouse sync, and it's ENR using https://enr-viewer.com/ )

**Pull request merged**